example/public/robots.txt

```diff
@@ -1,3 +1,3 @@
-# https://www.robotstxt.org/robotstxt.html
-User-agent: *
-Disallow: /
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow: /

```

Comment:
For example

Reply:
Line ending issue was addressed at least on my end.

Reply:
If it is not line ending issue, why is this change occurring? It seems like these files should not need to be deleted and then added in an identical way.